### DISCUSSION TITLE
#read-the-description empty

### DISCUSSION THREAD

**Discussion by CRISPYBIGMAC** [Monika-After-Story/MonikaModDev#8411] [QA - tech support + general] [answered]
When I PRESS #read the description, there is nothing. Soooooo... Could the maker fix that?

**Comment by ThePotatoGuy** [comment]
where are you clicking this?

**Comment by CRISPYBIGMAC** [comment]
in discord

**Comment by ThePotatoGuy** [comment, marked as answer]
You need to read the pins and the rules to gain access to the server.
